assets/js/vue/form-fields.js

```diff
@@ -55,23 +56,23 @@ export default {
     <template v-if="field.id !== '_gotcha'">
       <div :class="classList">
         <template v-if="field.type === 'textarea'">
-          <label class="input-label" :for="fieldId">{{ field.label }}</label>
-          <textarea class="form-control form-textarea" :id="fieldId" :name="fieldId" rows="4" :placeholder="placeholder" :required="required" :readonly="readonly"></textarea>
+          <label class="input-label" :for="setFieldId()">{{ field.label }}</label>
```

Comment:
kann man das nicht doch in einer computed property machen und dort die conditional logik mit reinnehmen? so wird die funktion teilweise pro feld 3x aufgerufen und das mit dem selben ergebnis